docs/plugins/integrating-search-into-plugins.md

```diff
@@ -333,3 +333,85 @@ reading the search context.
 If you produce something generic and reusable, consider contributing your
 component upstream so that all users of the Backstage Search Platform can
 benefit. Issues and pull requests welcome.
+
+#### Custom search results
+
+Search results throughout Backstage are rendered as lists so that list items can easily be customized; although a default result list item is available, plugins are in the best position to provide custom result list items that surface relevant information only known to the plugin.
```

Comment:
> a default result list item is available

maybe link to storybook?

Reply:
Good idea!  https://github.com/backstage/backstage/pull/15361/commits/9da74dcee627826c310c63e28065b1d1099fb035